doc/integrations/drupal8.rst

```diff
@@ -1,15 +1,19 @@
-Drupal 8
-========
+Drupal 8+
+=========
 
-Inc Files
-^^^^^^^^^
+Inc and module files
+^^^^^^^^^^^^^^^^^^^^
 
-By default Phpactor will not index `.inc` files.
+By default Phpactor will not index `.inc` nor `.module` files.
 
-Run the following on your project to enable the indexing of `.inc` files.
+Run the following on your project to enable the indexing of `.inc` and `.module` files.
 
 ```
-phpactor config:set indexer.supported_extensions '["php", "inc"]'
+phpactor config:set indexer.supported_extensions '["php", "inc", "module"]'
+```
+
+```
+phpactor config:set indexer.include_patterns '["/**/*.php", "/**/*.inc", "/**/*.module"]'
```

Comment:
hmm, I think I fixed this in the wrong way :flushed: with the current logic we first reduce the set of files to those with the "supported_extensions" and then again with the "include_patterns", so the "supported_extensions" wasn't strictly necessary.